tox.ini

```diff
@@ -25,7 +25,7 @@ commands =
     poetry run python -m pip install file://{toxinidir}/plugins/aea-ledger-fetchai
     poetry run python -m pip install file://{toxinidir}/plugins/aea-cli-ipfs
     poetry run python ./install_packages.py black isort
-    poetry run pytest -rfE tests/ --cov=aea --cov=packages/fetchai/connections --cov=packages/fetchai/contracts --cov=packages/fetchai/protocols --cov=packages/fetchai/skills --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing --cov-config=.coveragerc {posargs}
+    poetry run pytest -rfE --cov-config=setup.cfg --cov=aea --cov-report=html --cov-report=xml --cov-report=term --cov-report=term-missing {posargs}
```

Comment:
Why `cov-config=setup.cfg`? There is no config there for tests/coverage.

Reply:
Why take out coverage from `--cov=packages/fetchai/connections --cov=packages/fetchai/contracts --cov=packages/fetchai/protocols --cov=packages/fetchai/skills`?

Reply:
allpackage related cov options provided in workflow for packages-test
setup.cfg was applied before your changes

Reply:
done